docs/Overview.rst

```diff
@@ -36,17 +36,67 @@ MVS Validation Plan
 
-*Operational validity* determines if the model’s output is within the required accuracy. In order to achieve that, several validation techniques are used, namely:
+*Operational validity* assesses the model’s output with respect to the required accuracy. In order to achieve that, several validation techniques are used, namely:
 
-*	Benchmark testing, through which scenarios are created with different constraints and component combinations, and the output is calculated and compared to the expected one;
+* Graphical display, which is the use of model generated or own graphs for result interpretation. Graphs are simultaneously used with other validation techniques to inspect the results;
+
+*	Benchmark testing, through which scenarios are created with different constraints and component combinations, and the output is calculated and compared to the expected one to evaluate the performance of the model;
   
-*	Extreme scenarios (e.g., drastic meteorological conditions), created to make sure the simulation is through, then evaluate the output behavior by the use of graphs and qualitative analysis;
+*	Extreme scenarios (e.g., drastic meteorological conditions, very high costs, etc.), created to make sure the simulation is through and check if the output behavior is still valid by the use of graphs and qualitative analysis;
   
-*	Replication of one pilot project, with a validated optimization model with different component representation to compare and assess the results; and
+*	Comparison to other validated model, which replicates one pilot project into a validated optimization model with different component representation to identify the similarities and differences in results; and
   
 *	Sensitivity analysis, through which input-output transformations are studied to show the impact of changing the values of some input parameters.
 
+Implemented Tests
+#################
+
+Unit tests and integration tests are gauged by using test coverage measurement. The MVS covers so far 80% of the modules and sub-modules as seen in the next figure. Since those tests are automated, this coverage is updated for any changes in the model. The figure also shows that the build is currently passing.
+
+.. image:: images/Test_coverage.png
+ :width: 200
+
+A benchmark is a point of reference against which results are compared to assess the operational validity of a model. Benchmark tests are not automated like unit and integration tests, hence it is necessary to check that they are passing for any implemented changes in the model. The following table lists the implemented benchmark tests which cover several features and functionalities of the MVS.
+
+.. list-table:: Benchmark Tests
```

Comment:
Please add a column where you link to the input folders of the tests in the github repro

Reply:
You can do that by adding a hyperlink to the entries of the first column

Reply:
The table does not look nice in the read the docs: 
![grafik](https://user-images.githubusercontent.com/44204527/96471260-6dcf5800-122f-11eb-848a-b713f5d9e88c.png) 
Maybe an enumeration would be more beautiful (and readable)?

Reply:
https://multi-vector-simulator--593.org.readthedocs.build/en/593/Overview.html#id1

Reply:
Are these descriptions also in the py files of the benchmark tests? I am wondering whether it would help to link the appropriate pytests here as well. Suggestion:

> * **Electrical grid + PV** ([data](link to test data)/[pytest](link to pytest)): Maximum use of PV to serve the demand and the rest is compensated from the grid

Also, please replace "electrical grid" with "electricity grid".

Reply:
Note: Links are not added like I typed above, that was macro format.

Reply:
Hi @smartie2076 , do you have any hint on how to make those links work if this is not the correct way? I thought it's by including the relative directory path in the hyperlink

Reply:
Hi @ursulaelmir ! There are two ways for links - here is a hyperlink
> `this is printed in text <www.a_link.de>`_

You would then use a link to the test files in the master branch, ie. something in [here](https://github.com/rl-institut/multi-vector-simulator/tree/dev/tests/benchmark_test_inputs)

For references, you can look at the limitations part - I fixed the labels/references there (key: empty rows).